regparser/web/jobs/urls.py

```diff
@@ -3,8 +3,19 @@
 from rest_framework.urlpatterns import format_suffix_patterns
 
 urlpatterns = [
-    url(r'^job/$', views.JobViewList.as_view()),
-    url(r'^job/(?P<job_id>[-a-z0-9]+)/$',
+    url(r'^job(/)$', views.JobViewList.as_view()),
```

Comment:
I suspect this is due to some Django flag I haven't set, but locally, without that, navigating to e.g. `job/pipeline/` would get passed to the instance view as having a `job_id` of `None`; I want both `job/pipeline` and `job/pipeline/` to go to the list view.